get-cards.py

```diff
@@ -108,11 +105,11 @@ def main() :
         labels = ''
         for lab in dic_wekan['cards'][ v ]['labels'] :
             if labels == '' :
-                labels = dic_wekan['labels'][ lab ]['name']
+				labels = dic_wekan['labels'][ lab ]['name']
```

Comment:
Are you sure, that you want to use tabs?! This indentation doesn't look right.

Reply:
You're right sorry I'm fixing